aws_lambda_powertools/utilities/data_classes/common.py

```diff
@@ -1,9 +1,9 @@
 import base64
 import json
-from typing import Any, Dict, Optional
+from typing import Any, Dict, Iterator, Mapping, Optional
```

Comment:
```suggestion
from collections.abc import Mapping
from typing import Any, Dict, Iterator, Optional
```

Because your intent is to apply the protocol.